packages/server/lib/socket-base.ts

```diff
@@ -125,7 +125,7 @@ export class SocketBase {
       },
       destroyUpgrade: false,
       serveClient: false,
-      transports: ['websocket'],
+      transports: ['polling'],
```

Comment:
If we make this `['polling', 'websocket']`, then we can conditionally use what we need depending on the browser, though I think we should try and understand why it's not working in the first-place and see if there are any workarounds.

Reply:
Yeah, I was thinking about this. My understanding is that the main issue that Webkit doesn't proxying the web socket traffic and it appears that Cypress demand this happens. The main way to fix this would be trying to create a repro for the Playwright team so they could fix it. Only I have been waiting for some feedback on this PR before working on that. 

If anyone of Cypress have a such a repro (or basis for it) lying around that would be great! :D